sktime/classification/deep_learning/gru.py

```diff
@@ -0,0 +1,429 @@
+"""Gated Recurrent Unit (GRU) for time series classification."""
+
+import numpy as np
+
+from sktime.classification.deep_learning._pytorch import BaseDeepClassifierPytorch
+from sktime.utils.dependencies import _check_soft_dependencies
+
+if _check_soft_dependencies("torch", severity="none"):
+    import torch
+
+    from sktime.networks.gru import GRU, GRUFCNN
+
+    OPTIMIZERS = {
```

Comment:
should this be in a more central location perhaps? These are very general, so could be put as a (private) method into the torch base class.

Reply:
Yeah, this can be moved to the base class.